src/internal/modules/abstraction-reform/lib/action-creators.js

```diff
@@ -92,7 +92,7 @@ const createEditCondition = (data, defra, id) => {
  * @param {Object} user - the current user of the application
  * @return {Object} action to edit condition
  */
-const createSetStatus = (status, notes, defra) => {
+const createSetStatus = (status, notes, user) => {
```

Comment:
You changed defra to user here but in the rest of the file it remains as defra?
Perhaps this line should change instead
`@param {Object} user - the current user of the application` to 
`@param {Object} defra - the current user of the application`

either way no big deal but probably best to be consistent in the file